### DISCUSSION TITLE
Slash Command Cooldown

### DISCUSSION THREAD

**Discussion by Mehul343** [nextcord/nextcord#366] [Q&A]
How to add cooldown to slash commands?

**Comment by Soheab** [comment]
Discord cooldowns the user, why do you need this?

**Reply by VincentRPS** [reply]
Probably is useful for economy commands

**Reply by Mehul343** [reply]
> Discord cooldowns the user, why do you need this?

I need a custom cooldown like an 8 second cooldown

**Reply by Mehul343** [reply]
> Probably is useful for economy commands

Yes

**Reply by Soheab** [reply]
Aah I didn't think of those types of commands.
